optimize/client/src/modules/components/ReportRenderer/HyperReportRenderer.test.js

```diff
@@ -89,66 +105,28 @@ it('should render single process report if report result is empty', () => {
     />
   );
 
-  expect(node).toMatchSnapshot();
+  expect(node.find(ProcessReportRenderer)).toExist();
 });
 
-it('should format single reports names for hyper reports distributed by date variable', () => {
-  const node = shallow(
-    <HyperReportRenderer
-      report={{
-        combined: false,
-        reportType: 'process',
-        data: {
-          processDefinitionKey: 'aKey',
-          processDefinitionVersion: '1',
-          view: {
-            properties: ['frequency'],
-            entity: 'processInstance',
-          },
-          groupBy: {
-            type: 'startDate',
-          },
-          visualization: 'bar',
-          distributedBy: {
-            type: 'variable',
-            value: {type: 'Date'},
-          },
-          configuration: {
-            distributeByDateVariableUnit: 'month',
-          },
-        },
-        result: {
-          measures: [
-            {
-              property: 'frequency',
-              data: [
-                {
-                  key: '2020-09-01T00:00:00.000+0200',
-                  label: '2020-09-01T00:00:00.000+0200',
-                  value: [
-                    {
-                      key: '1969-12-07T14:52:00Z',
-                      value: 1.0,
-                      label: '1969-12-07T14:52:00Z',
-                    },
-                    {
-                      key: '2020-03-08T14:52:00Z',
-                      value: 1.0,
-                      label: '2020-03-08T14:52:00Z',
-                    },
-                  ],
-                },
-              ],
-            },
-          ],
-          instanceCount: 1234,
-          type: 'hyperMap',
-        },
-      }}
-    />
-  );
+it('should convert adjust the result to be identical to hyper report result structure', () => {
```

Comment:
I don't understand the description here
```suggestion
it('should convert and adjust the result to be identical to hyper report result structure', () => {
```